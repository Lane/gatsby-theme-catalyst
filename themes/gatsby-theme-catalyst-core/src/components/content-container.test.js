import React from "react"
import renderer from "react-test-renderer"
import ContentContainer from "./content-container"

describe("ContentContainer", () => {

  const props = {
    children: 'test'
  }

  it("renders correctly", () => {
    const tree = renderer
      .create(<ContentContainer {...props} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})