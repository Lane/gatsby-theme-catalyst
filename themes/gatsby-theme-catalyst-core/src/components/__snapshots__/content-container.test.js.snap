// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentContainer renders correctly 1`] = `
<div
  className="css-1gjvup9-ContentContainer"
>
  test
</div>
`;
